api/teaser/[articleId].js

```diff
@@ -0,0 +1,16 @@
+import connectToMongodb from '../../backend/db/connect';
+import Blogposts from '../../backend/models/Blogposts';
+
+export default async function handler(request, response) {
+	await connectToMongodb();
+
+	const {articleId} = request.query;
+	const post = await Blogposts.findById(articleId);
+
+	switch (request.method) {
```

Comment:
Why are you using a switch here?  🤔 `try catch` is more common.

Reply:
later, there will also be delete method for specific posts, so in advance I chose to use switch. So I don't have to change it later.